pkg/logs/config/config.go

```diff
@@ -135,8 +135,9 @@ func buildTCPEndpoints() (*Endpoints, error) {
 	useProto := coreConfig.Datadog.GetBool("logs_config.dev_mode_use_proto")
 	proxyAddress := coreConfig.Datadog.GetString("logs_config.socks5_proxy_address")
 	main := Endpoint{
-		APIKey:       getLogsAPIKey(coreConfig.Datadog),
-		ProxyAddress: proxyAddress,
+		APIKey:                  getLogsAPIKey(coreConfig.Datadog),
+		ProxyAddress:            proxyAddress,
+		ConnectionResetInterval: time.Duration(coreConfig.Datadog.GetInt("logs_config.connection_reset_interval")) * time.Second,
```

Comment:
We have a `GetDuration` method for this.

Reply:
`GetDuration` expects the option to be defined as a duration-formation string, whereas this option is defined as a integer representing seconds at the moment, so we can't use it